lib/sgov/entity-adapters/sgov-entity-adapter.ts

```diff
@@ -0,0 +1,30 @@
+import {RdfObject, RdfSourceWrap} from "../../core/adapter/rdf";
+import {PimResource} from "../../pim/model";
+import {LanguageString} from "../../core";
+import {SKOS} from "../sgov-vocabulary";
+import {IriProvider} from "../../cim/iri-provider";
+
+// todo move
+function RdfObjectsToLanguageString(objects: RdfObject[]): LanguageString {
+  return Object.fromEntries(objects.map(o => [o.language, o.value]));
+}
+
+export async function loadSgovEntity(entity: RdfSourceWrap, idProvider: IriProvider): Promise<PimResource> {
+  const resource: PimResource = {types: [], iri: null};
+
+  // skos:prefLabel
```

Comment:
Are those comments really adding any value? Like the fact that on the next line we work with SKOS.prefLabel is obvious from the code it self.